clients/fides-js/src/lib/tcf/renderOverlay.tsx

```diff
@@ -0,0 +1,7 @@
+import { ContainerNode, render, h } from "preact";
+import TcfOverlay from "../../components/tcf/TcfOverlay";
+import { OverlayProps } from "../../components/types";
+
+export const renderOverlay = (props: OverlayProps, parent: ContainerNode) => {
+  render(<TcfOverlay {...props} />, parent);
```

Comment:
this file is the same as `lib/renderOverlay` except we return the `TcfOverlay` component instead of the `NoticeOverlay` component